packages/main/src/Label.js

```diff
@@ -71,6 +71,12 @@ const metadata = {
 		"for": {
 			type: String,
 		},
+
+		_tabIndex: {
```

Comment:
For this changes in the Label, you would need to inform the colleagues from @SAP/ui5-webcomponents-topic-rd  and let them know the use case